src/common/transport/serialization_transport.cpp

```diff
@@ -160,12 +160,15 @@ uint32_t SerializationTransport::send(uint8_t *cmdBuffer, uint32_t cmdLength, ui
 // Event Thread
 void SerializationTransport::eventHandlingRunner()
 {
-    while (runEventThread) {
-
+    do {
```

Comment:
Why change to _do_ instead of _while_ here?

Reply:
You are right, it doesn't make much difference. It was only the order of my logic as I think about that as exit condition not as entry condition. I change it back.